<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainSettingWindow</class>
 <widget class="QMainWindow" name="MainSettingWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1153</width>
    <height>704</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainSettingWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_3" rowstretch="0" columnstretch="1,1" columnminimumwidth="1,1">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <property name="spacing">
       <number>5</number>
      </property>
      <item row="0" column="0">
       <widget class="QGraphicsView" name="graphicsView"/>
      </item>
      <item row="0" column="1">
       <layout class="QVBoxLayout" name="verticalLayout" stretch="1,2,1,1,1,6,1,1">
        <property name="leftMargin">
         <number>4</number>
        </property>
        <property name="topMargin">
         <number>4</number>
        </property>
        <property name="rightMargin">
         <number>4</number>
        </property>
        <property name="bottomMargin">
         <number>4</number>
        </property>
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>颜色选择:</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="7">
           <widget class="QPushButton" name="pushButton_7">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="QPushButton" name="pushButton_4">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="pushButton_2">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="5">
           <widget class="QPushButton" name="pushButton_5">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="pushButton_1">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="6">
           <widget class="QPushButton" name="pushButton_6">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QPushButton" name="pushButton_3">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QPushButton" name="pushButton_0">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="pushButton_8">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="pushButton_9">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="pushButton_10">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QPushButton" name="pushButton_11">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="4">
           <widget class="QPushButton" name="pushButton_12">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="5">
           <widget class="QPushButton" name="pushButton_13">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="6">
           <widget class="QPushButton" name="pushButton_14">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="7">
           <widget class="QPushButton" name="pushButton_15">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>闪烁模式：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_2">
            <item>
             <property name="text">
              <string>常亮</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>闪烁</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>简便</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>流水</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>持续时间：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox">
            <item>
             <property name="text">
              <string>21ms</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>5ms</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>10ms</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>20ms</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>50ms</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>100ms</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>150ms</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>200ms</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>500ms</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1s</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2s</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>5s</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>10s</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>20s</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>50s</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1min</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>选择产品：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QScrollArea" name="scrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>550</width>
             <height>267</height>
            </rect>
           </property>
           <widget class="QWidget" name="gridLayoutWidget_2">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>641</width>
              <height>381</height>
             </rect>
            </property>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="0">
              <widget class="myLabel" name="label_3">
               <property name="text">
                <string>TextLabel</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="myLabel" name="label_7">
               <property name="text">
                <string>TextLabel</string>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="myLabel" name="label_5">
               <property name="text">
                <string>TextLabel</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="myLabel" name="label_6">
               <property name="text">
                <string>TextLabel</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="myLabel" name="label_4">
               <property name="text">
                <string>TextLabel</string>
               </property>
              </widget>
             </item>
             <item row="2" column="2">
              <widget class="myLabel" name="label_8">
               <property name="text">
                <string>TextLabel</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>下发数据</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_18">
          <property name="text">
           <string>上传数据</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1153</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>myLabel</class>
   <extends>QLabel</extends>
   <header location="global">mylabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
